examples/cluster/main.tf

```diff
@@ -0,0 +1,33 @@
+terraform {
+  required_providers {
+    dfcloud = {
+      source = "registry.terraform.io/dragonflydb/dfcloud"
+    }
+  }
+}
+
+provider "dfcloud" {
+}
+
+resource "dfcloud_datastore" "cache-cluster" {
+  name = "frontend-cache-cluster"
+
+  location = {
+    region   = "us-central1"
+    provider = "gcp"
+  }
+
+  tier = {
+    max_memory_bytes = 6000000000
+    performance_tier = "dev"
+    replicas         = 1
+  }
+
+  cluster = true
```

Comment:
should we group cluster and shard memory together in a cluster = { } config ?

Reply:
what happens if I change cluster form true to false in the TF script  and apply?

Reply:
> trying to keep everything as flat as possible

Why? Isn't it inconsistent with e.g tier